class Login_Error(Exception):
    pass